golang/cosmos/types/address_hooks_test.go

```diff
@@ -86,32 +68,37 @@ func TestExtractBaseAddressFromPacket(t *testing.T) {
 	channeltypes.RegisterInterfaces(ir)
 	clienttypes.RegisterInterfaces(ir)
 
+	cosmosHook, err := types.JoinHookedAddress("cosmos1qqxuevtt", []byte("?foo=bar&baz=bot#fragment"))
+	require.NoError(t, err)
```

Comment:
This seems to test only that the call does not err. Is that right?

I'm curious what happens to `#fragment`.

Reply:
I tested SplitHookedAddress and verified the results.  That turned up a bug (wasn't stripping the baseAddr length bytes from the end of an Address Hook).  Thanks!